orbs/circleci/circle-wait-job

```diff
@@ -99,83 +99,96 @@ uri = URI::HTTPS.build(host: "circleci.com",
                                                    "shallow" => true,
                                                    "filter" => "running"}))
 
+uri_for_successful = URI::HTTPS.build(host: "circleci.com",
+                       path: File.join("/",
+                                       "api",
+                                       "v1.1",
+                                       "project",
+                                       options.vcs.to_s,
+                                       options.organization.to_s,
+                                       options.project.to_s,
+                                       "tree",
+                                       options.branch.to_s),
+                       query: URI.encode_www_form({"circle-token" => options.token.to_s,
+                                                   "shallow" => true,
+                                                   "filter" => "successful"}))
```

Comment:
**remark**

Since `uri_for_successful` is 99% similar to `uri_for_running`, you could define a function to build both, passing the `filter` as param to the function.